src/routes/volume/VolumeActions.js

```diff
@@ -5,6 +5,42 @@ import { DropOption } from '../../components'
 const confirm = Modal.confirm
 
 function actions({ selected, engineImages, showAttachHost, detach, showEngineUpgrade, deleteVolume, showBackups, showSalvage, rollback, showUpdateReplicaCount, showExpansionVolumeSizeModal, showCancelExpansionModal, createPVAndPVC, changeVolume, confirmDetachWithWorkload, commandKeyDown }) {
+  const deleteWranElement = (record) => {
+    let workloadStatus = ''
+    let hasPvTootipText = ''
+    let pvStatus = ''
+
+    if (record && record.kubernetesStatus && record.kubernetesStatus.pvStatus) {
+      hasPvTootipText = (<div>{`The longhorn-manager will delete the Kubernetes resources associated with the Volume ${record.name}`}</div>)
+      pvStatus = (<div style={{ marginTop: '10px', color: '#b7b7b7' }}>
+        <div style={{ paddingLeft: '10px' }}>PV Name: {record.kubernetesStatus.pvName}</div>
+        <div style={{ paddingLeft: '10px' }}>PVC Name: {record.kubernetesStatus.pvcName}</div>
+        <div style={{ paddingLeft: '10px' }}>PV Status: {record.kubernetesStatus.pvStatus}</div>
+      </div>)
+    }
+    if (record.kubernetesStatus && record.kubernetesStatus.workloadsStatus && record.kubernetesStatus.workloadsStatus.length > 0) {
+      workloadStatus = (<div>
+        <div style={{ marginTop: '10px', color: '#b7b7b7' }}>{record.kubernetesStatus.workloadsStatus.map((item, i) => {
+          return (<div key={i} style={{ paddingLeft: '10px' }}>Pod Name: {item.podName} ({item.podStatus})</div>)
+        })}</div>
+      </div>)
+    }
+    if (record.state === 'attached') {
+      return (<div>
+          <div>{`The Volume is Attached on ${record.controllers && record.controllers[0] && record.controllers[0].hostId ? record.controllers[0].hostId : ''} and that deleting the Volume can lead to errors in any applications using the Volume`} </div>
```

Comment:
I think this can be rephrased to:
```
The volume is attached on ... Deleting this volume may cause errors for any running applications using this volume!
```